doctr/models/detection/differentiable_binarization/base.py

```diff
@@ -132,6 +134,38 @@ def bitmap_to_boxes(
                 x, y, w, h = x / width, y / height, w / width, h / height
                 boxes.append([x, y, w, h, alpha, score])
 
+        if not self.assume_straight_pages:
+            # Compute median angle and mean aspect ratio to resolve quadrants
+            median_angle = np.median(np.float32(boxes)[:, -2])
+            mean_w = np.mean(np.float32(boxes)[:, 2])
+            mean_h = np.mean(np.float32(boxes)[:, 3])
```

Comment:
uncommon way to cast the dtype :sweat_smile: 
Perhaps:
```python
np_boxes = np.asarray(boxes, dtype=np.float32)
median_angle, mean_w, mean_h = np.median(np_boxes[:, -2]), np_boxes[:, 2].mean(), np_boxes[:, 3].mean()
```

Also shouldn't we use median instead of mean for h & w ?

Reply:
I don't know, I think the distribution is much more homogeneous than the angle so I thought we could keep the mean h & mean w

Reply:
Might be better to use the median just in case :+1: